doc/userguide/src/Appendices/CommandLineOptions.rst

```diff
@@ -48,6 +49,7 @@ Command line options for test execution
                            *Statistics by Suite* table in outputs.
   --tagstatinclude <tag>  `Includes only these tags`_ in the *Statistics by Tag* table.
   --tagstatexclude <tag>  `Excludes these tags`_ from the *Statistics by Tag* table.
+  --tagstatskip <tag>     `Skips these tags`_ from the *Statistics by Tag* table.
```

Comment:
First off, sorry Pekkaklarck for taking so long to get back to you on this, things have been hecktic at work of late.

My implementation of --tagstatskip is exactly the same as --tagstateclude.

With reference to your comment on 16th Oct https://github.com/robotframework/robotframework/issues/2087

"If skipping is supported, I think it should also be supported based on tags e.g. like --skip mytag. Tests having skipped tags shouldn't be executed at all, and a dynamically added skip tag should mark test skipped afterwards. In practice --skip would work like --exclude but tests would be shown in statistics."

My understanding of teh robot source is limited and I could not see exactly how to implement
what you were asking for in yor comment. So Implemented as much as I could in replicating
--tagstatexclude with --tagstatskip. I was hoping that you (or someone else in the robot
development could/would use what I have writtn as a basses for what you are asking for. I don't
understand teh source enough to do this correctly I feel :(